samples/Azure.Management.Storage/Generated/BlobContainer.cs

```diff
@@ -22,6 +22,12 @@ namespace Azure.Management.Storage
     /// <summary> A Class representing a BlobContainer along with the instance operations that can be performed on it. </summary>
     public partial class BlobContainer : ArmResource
     {
+        /// <summary> Generate the resource identifier of a <see cref="BlobContainer"/> instance. </summary>
+        public static ResourceIdentifier BuildId(string subscriptionId, string resourceGroupName, string accountName, string containerName)
```

Comment:
I wonder if we should call this CreateResourceIdentifier

Reply:
@m-nash I'm fine with the change. `BuildId` is from original discussion thread. I was also wondering if we should use another name.